packages/editor/src/components/post-title/index.js

```diff
@@ -221,15 +218,33 @@ function PostTitle( _, forwardedRef ) {
 				};
 			} );
 		},
-		__unstableDisableFormats: true,
+		__unstableDisableFormats: false,
 		preserveWhiteSpace: true,
 	} );
 
+	const richEditorRef = useMergeRefs( [ richTextRef, ref ] );
+
+	// Setting to "2" will not render raw HTML.
+	const plainTextVersion = 1;
```

Comment:
@ellatrix I don't know why but using v2, caused the HTML not to render _at all_. The tags were stripped from the content. Whereas in v1, the tags are preserved in raw form (which is what we want in "Code view" mode).

Reply:
PlainText is not really meant to be used outside of blocks btw. I'll have a look into why this happens though. If it's a bug, it should happens in some blocks